manual/object_mapper/using/README.md

```diff
@@ -297,3 +297,49 @@ public ListenableFuture<Result<User>> getAllAsync();
 ```
 
 [@QueryParameters]: http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/mapping/annotations/QueryParameters.html
+
+
+### Mapping configuration
```

Comment:
This is probably the right amount of documentation 👍 .  I think if the user wants to go beyond this they probably need to take time to understand the code.   We can always add more based on questions we get from community.